pydantic/env_settings.py

```diff
@@ -88,3 +103,19 @@ def prepare_field(cls, field: ModelField) -> None:
             field.field_info.extra['env_names'] = env_names
 
     __config__: Config  # type: ignore
+
+
+def read_env_file(file_path: Path, case_sensitive: bool = False) -> Dict[str, str]:
+    file_vars: Dict[str, str] = {}
+    for n, line in enumerate(re.split('[\r\n]+', file_path.read_text()), start=1):
+        line = line.strip()
+        if line and not line.startswith('#'):
+            m = re.match(r'^\s*(\w+)\s*=\s*(.+?)\s*$', line)
```

Comment:
```suggestion
            m = re.match(r'^(?:export )?\s*(\w+)\s*=\s*(.+?)\s*$', line)
```
Just occurred to me, as with python-dotenv we should allow and ignore lines starting with `export ` so `.env` files also work with `source`.